tests/_files/test_vehicle_ts/res_SimBEV_Berlin_2015.csv

```diff
@@ -0,0 +1,13 @@
+;time;sum CS power;sum UC work;sum UC business;sum UC school;sum UC shopping;sum UC private/ridesharing;sum UC leisure;sum UC home;sum UC hub
+0;01.01.2015 00:00;6011,442277;113,7397321;0;0;22;132,2632421;813,1647798;4810,274522;120
+1;01.01.2015 00:15;5834,261506;43,29294752;0;0;0;245,5309077;664,090325;4881,347325;0

```

Comment:
The test data could have more intuitive values, to be able to check the result manually. I haven't done this, yet, as I remember that you were not sure about keeping all tests of single functions @jnnr.